Tests/SwiftGenKitTests/StringsTests.swift

```diff
@@ -118,10 +118,16 @@ class StringsTests: XCTestCase {
     XCTAssertEqual(placeholders, [.unknown, .int, .object, .float, .char])
   }
 
-  func testParseEscapePercentSign() {
+  func testParseEvenEscapePercentSign() {
     let placeholders = StringsFileParser.PlaceholderType.placeholders(fromFormat: "%%foo")
     // Must NOT map to [.Float]
     XCTAssertEqual(placeholders, [])
   }
 
+  func testParseOddEscapePercentSign() {
+    let placeholders = StringsFileParser.PlaceholderType.placeholders(fromFormat: "%%%foo")
+    // Should map to [.Float]
+    XCTAssertEqual(placeholders, [.Float])
```

Comment:
Right, so your branch was based on an older version of SwiftGenKit. This needs to be `.float` instead.